doc/OnlineDocs/modeling_extensions/gdp.rst

```diff
@@ -61,32 +60,36 @@ the example returns.
 Transformation
 --------------
 
-In order to use the solvers currently available, one must convert the
-disjunctive model to a standard MIP/MINLP model.  The easiest way to do
-that is using the (included) BigM or Convex Hull transformations.  From
-the Pyomo command line, include the option ``--transform
-pyomo.gdp.bigm`` or ``--transform pyomo.gdp.chull``
+To use standard commercial solvers, you must convert the disjunctive model to a standard MIP/MINLP model.
+The two classical strategies for doing so are the (included) Big-M and Hull reformulations.
+From the Pyomo command line,  include the option ``--transform pyomo.gdp.bigm`` or ``--transform pyomo.gdp.chull``.
+If you are using a Python script, ``TransformationFactory`` accomplishes the same functionality:
 
-Notes
------
+- ``TransformationFactory('gdp.bigm').apply_to(model)``
+- ``TransformationFactory('gdp.chull').apply_to(model)``
+
+.. note::
 
-Some notes:
+    - all variables that appear in disjuncts need upper and lower bounds
 
-- all variables that appear in disjuncts need upper and lower bounds
+    - for linear models, the BigM transform can estimate reasonably tight M
+      values for you
 
-- for linear models, the BigM transform can estimate reasonably tight M
-  values for you
+    - for all other models, you will need to provide the M values through a
+      “BigM” Suffix.
 
-- for all other models, you will need to provide the M values through a
-  “BigM” Suffix.
+    - When you declare a Disjunct, it (at declaration time) will automatically
+      have a variable “indicator_var” defined and attached to it.
+      After that, it is just a Var like any other Var.
 
-- the convex hull reformulation is only valid for linear and convex
-  nonlinear problems.  Nonconvex problems are not supported (and are not
-  checked for).
```

Comment:
Why are you removing this warning?

Reply:
I might have some results here that I would be happy to talk to you about. Even in that case, I would just leave the warning (this might go back to the whole convex hull vs. hull reformulation debate)